### DISCUSSION TITLE
Debian Package qdirstat replaces over shortcut super+e dde-file-manager after install

### DISCUSSION THREAD

**Discussion by JohnML1** [linuxdeepin/developer-center#3001] [Q&A | 问答板块] [answered]
A) I don't know where  exact to report this issue ...

B) I'm on Deepin 20.1, all updates applied.

Try this: apt install qdirstat
and then open dde-file-manager per default shortcut super+e

I get unwanted qdirstat instead of dde-file-manager

If you delete qdirstat.desktop it behaves as aspected.

**Comment by juteman** [comment, marked as answer]
This is  shortcut key conflict, super+e is setting shortcut by many application such as vscode. 
you can use this command `gio mime inode/directory dde-file-manager.desktop` to set super+e defaut as dde-file-manager
